apps/phone/src/apps/contacts/components/views/ContactInfo.tsx

```diff
@@ -81,10 +159,8 @@ const ContactsInfoPage: React.FC = () => {
 
   return (
     <div className="mx-auto h-full w-full">
-      <button
-        onClick={() => history.goBack()}
-        className="mt-4 ml-4 rounded-md px-3 py-1 hover:bg-gray-200 hover:dark:bg-neutral-800"
-      >
+      <SendMoneyModal open={contactPayModal} closeModal={() => setcontactPayModal(false)} openContact={number} />
```

Comment:
This should probably be added as a config for the people who enable this integration

Reply:
it already is